de.metas.shipper.gateway.dhl/src/main/java/de/metas/shipper/gateway/dhl/model/DhlCustomDeliveryDataDetail.java

```diff
@@ -61,5 +69,13 @@ private DhlCustomDeliveryDataDetail(
 		this.pdfLabelData = pdfLabelData;
 		this.awb = awb;
 		this.trackingUrl = trackingUrl;
+		this.internationalDelivery = internationalDelivery;
+//		if (internationalDelivery && customsDocument == null)
```

Comment:
again, please clean this up, or add a comment explaining the use of this commented out code